### PR TITLE
4.2.0-beta1 docs work

### PR DESCRIPTION
**Pull request by rozza**
http://rozza.github.io/mongo-java-driver/

### CODE REVIEW

File: docs/reference/content/index.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ type = "index"
 
 ## MongoDB Java Driver Documentation
 
-Welcome to the MongoDB JVM driver documentation hub for the 4.1 drivers release.
+Welcome to the MongoDB JVM driver documentation hub for the 4. 2 drivers release.
```
Comment (jyemin) [inline docs/reference/content/index.md:9]: remove extra space

File: docs/reference/content/whats-new.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ title = "What's New"
   pre = "<i class='fa fa-level-up'></i>"
 +++
 
+# What's new in 4.2
+
+This release fully supports all MongoDB releases from versions 2.6 to 4.4. Key new features of the 4.2 Java driver release include:
+
+* Client-side field level encryption has been expanded to support Azure and GCP Keystores.
```
Comment (jyemin) [inline docs/reference/content/whats-new.md:14]: can  you add a sentence about the kerberos update in 3836?

**Review by jyemin — CHANGES_REQUESTED** [review]
There are a couple places that mention AWS key provider as the only other option besides local.  Can you update those?

otherwise looks great

File: docs/reference/content/index.md (hunk 1 above)
Comment (rozza) [reply docs/reference/content/index.md]: Done

File: docs/reference/content/whats-new.md (hunk 2 above)
Comment (rozza) [reply docs/reference/content/whats-new.md:14]: Done

**Comment by jyemin** [comment]
can you respond to my general comment above about AWS references?

**Comment by rozza** [comment]
I fixed the one place I saw it had been missed in the Scala driver, checking for others now

**Comment by rozza** [comment]
@jyemin I think it was just the scala encryption version that hadn't been updated.

**Review by jyemin — APPROVED** [review]

**Pull request closed without merging**